html-forms/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <form action="">
+      <h1>Coding Form!</h1>
+      <div>
+        <label for="email">Email</label>
+      </div>
+      <div>
+        <input type="email" name="email" id="email">
+      </div>
+      <div>
+        <label for="reason">Why is coding cool?</label>
+      </div>
+      <div>
+        <select name="reason">
+          <option value="option">Select an option</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <div>
+        <label for="language">Favorite Language (choose one)</label>
```

Comment:
in this case, we want to wrap a label around each of the radio inputs as we want to maintain label click functionality. This will be discussed further during Q And A